alias/dlgalias.c

```diff
@@ -239,6 +283,54 @@ static void dlg_select_alias(char *buf, size_t buflen, struct AliasMenuData *mda
           }
         }
         break;
+      case OP_SORT:
+      case OP_SORT_REVERSE:
+      {
+        int sort = C_SortAlias;
+        bool resort = true;
+        bool reverse = (op == OP_SORT_REVERSE);
+
+        switch (mutt_multi_choice(
+            reverse ?
+                /* L10N: The highlighted letters must match the "Sort" options */
+                _("Rev-Sort (a)lias, a(d)dress, (u)nsorted or d(o)n't sort?") :
+                /* L10N: The highlighted letters must match the "Rev-Sort" options */
+                _("Sort (a)lias, a(d)dress, (u)nsorted or d(o)n't sort?"),
```

Comment:
We don't want the ` or d(o)n't sort` any more.
The user can hit <kbd>a</kbd>, <kbd>d</kbd>, <kbd>u</kbd> or <kbd>Ctrl-G</kbd> to cancel.

Reply:
Done.